lang/test/org/partiql/lang/eval/SimpleEvaluatingCompilerTests.kt

```diff
@@ -124,12 +125,197 @@ class SimpleEvaluatingCompilerTests : EvaluatorTestBase() {
     }
 
     @Test
-    fun max() {
+    fun maxAndMin() {
```

Comment:
Sorry missed this from the previous review, but it can be annoying to test an individual query since all of these tests are in one test function. I'd suggest either using `ParameterizedTest` or breaking individual or closely-related tests into separate test functions.

Reply:
If we want to use `ParameterizedTest`, I think we should refactor the whole file to use that. However, I think this is outside the scope of this PR.